src/Eccube/Resource/template/default/Shopping/shipping_edit.twig

```diff
@@ -35,7 +35,13 @@ Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA  02111-1307, USA.
 {% endblock javascript %}
 
 {% block main %}
-    <h1 class="page-heading">お届け先の追加</h1>
+    <h1 class="page-heading">
+    {% if app.user is not null and is_granted('ROLE_USER') %}
```

Comment:
In symfony version >= 2.8, Check 'app.user' was included in the 'is_granted'.
But in symfony version <2.8 (eccube3), we have to handle it.

'App.user' including administrators and customers, so we have to check 'ROLE_USER'.